stable/awsebscsiprovisioner/templates/roles.yaml

```
@@ -187,6 +187,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots/status"]
     verbs: ["update"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["create", "list", "watch", "delete"]
```

```suggestion
    verbs: ["create", "list", "get"]
```

I disagree, somewhat, with the previous conversation. It's considered best practice to apply the [principal of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). If create is the minimum this controller needs, that's what it should get.

I suspect, however, that the controller would actually need list, get, and update in order to check to see if the crd is installed, check to see if it's current, and to update it if the installed crd is out of date. I doubt it uses watch and we don't want CRDs being deleted by automation.